src/Sql/Sql/ManagedInstance/Cmdlet/NewAzureSqlManagedInstance.cs

```diff
@@ -213,6 +213,13 @@ public class NewAzureSqlManagedInstance : ManagedInstanceCmdletBase
             HelpMessage = "Generate and assign an Azure Active Directory Identity for this instance for use with key management services like Azure KeyVault.")]
         public SwitchParameter AssignIdentity { get; set; }
 
+        /// <summary>
+        /// Gets or sets the managed instance compute generation
+        /// </summary>
+        [Parameter(Mandatory = false,
+            HelpMessage = "The Dns Zone Partner Resource ID for the Sql Azure Managed Instance.")]
+        public string DnsZonePartner { get; set; }
```

Comment:
Are there any known values for this, or is this always free-form text?

Reply:
There is no known value. it is an ARM resource id

Reply:
How does the user discover the value of this resourceId?  Seems like we should decorate this property with the ResourceId Completer: https://github.com/Azure/azure-powershell/blob/1bcbe7b1f7a3323ac98f7754ba03eeb6b45e79f2/src/Compute/Compute/VirtualMachine/Action/VirtualMachineActionBaseCmdlet.cs

Reply:
Added the decorator.